subjects/quarterofayear/README.md

```diff
@@ -0,0 +1,50 @@
+## quarterofayear
+
+### Instructions
+
+Write a function `QuarterOfAYear()` that takes an `int`, from 1 to 12, as an argument and returns an `int` with the respective quarter of the year to which it belongs.
+
+- If the number is not between 1 and 12 return `-1`.
+
+
+### Expected function
+
+```go
+func QuarterOfAYear(month int) int {
+
+}
+```
+
+### Usage
+
+Here is a possible program to test your function:
+
+```go
+package main
+
+import (
+	"fmt"
+	"piscine"
+)
+
+func main() {
+	fmt.Println(piscine.QuarterOfAYear(2))
+	fmt.Println(piscine.QuarterOfAYear(5))
+	fmt.Println(piscine.QuarterOfAYear(8))
+	fmt.Println(piscine.QuarterOfAYear(11))
```

Comment:
It would be nice to have an example of a month that lies in the 'frontier' between quarters, and less examples of similar cases, for instance, 2, 5 and 8, are all the second month of the quarter, it would be enough with only one